Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -206,12 +206,22 @@ void Settings::SetStateSlot(int slot)
 
 bool Settings::GetRenderToMain() const
 {
-  return value(QStringLiteral("Graphics/RenderToMain"), false).toBool();
+  return SConfig::GetInstance().bRenderToMain;
+}
+
+void Settings::SetRenderToMain(bool rendertomain)
```

Comment:
Why do all of these need to be in `Settings`, rather than accessed directly? Unless Settings is doing something special with them, like emitting signals, I don't see a need to add a layer of indirection here.